### PR TITLE
Add Frontend ReadME.md

### PR DESCRIPTION
**Pull request by Ethella**

### CODE REVIEW

File: Frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+libraries you use
+
+- **Technology** Choices: What are some of the tradeoffs that you made to pick the
+
+technologies (frameworks, libraries, etc.) you used?
+
+- **Code Quality** : Is the code easy to read and maintainable? Consistent styling and
+
+best practices for the languages used.
```
Comment (FYJen) [inline Frontend/README.md]: Nit: We probably don't need the newline here.

File: Frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- A README should be included. Treat it like a service that you are pushing to production.ins and outs of the service should be discussed
+
+- Should focus on making the interfaces as polished as possible
+- Make sure it runs on multiple browsers and platforms
+
+### **Code Aspects:**
+
+- **Architecture** : Code layout and architecture for the given frameworks and
+
+libraries you use
```
Comment (FYJen) [inline Frontend/README.md]: Nit: We probably don't need the newline here.

File: Frontend/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+## **Frontend Coding Challenge**
+
+### **General Guidelines:**
+
+- We value quality over quantity (feature-completeness). It is totally fine if things are left out as long as you make a note of it so we can see the trade-off and why you make the
+
+Decisions.
```
Comment (FYJen) [inline Frontend/README.md]: The Decision should be with the previous sentence :)

File: Frontend/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+- Requirements:
+  - It should be a client-side project. No backend server required
+  - A user should be able to search for locations
+  - Create an itinerary
+  - Add movie location(s) to the trip
+  - Remove movie location(s)
+  - The user&#39;s trip planner should be persisted
+  - A map should be used to display the locations
+  -
```
Comment (FYJen) [inline Frontend/README.md]: Remove the additional `-`

File: Frontend/README.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- Description:
+  - Creates an app that shows where movies have been filmed on a map in SF. A trip planner that collects movie locations for the trip.
+
+- Requirements:
+  - It should be a client-side project. No backend server required
+  - A user should be able to search for locations
+  - Create an itinerary
+  - Add movie location(s) to the trip
+  - Remove movie location(s)
+  - The user&#39;s trip planner should be persisted
```
Comment (FYJen) [inline Frontend/README.md]: Let's not encode the `'`

**Review by FYJen — COMMENTED** [review]
Just some nits. I will do another pass after.

**Comment by Ethella** [comment]
[ch23438]

**Review by FYJen — APPROVED** [review]

**Pull request merged**